### ISSUE TITLE
Attempting to make Tile Map in background, getting visual issues

### ISSUE THREAD

**Issue by krazbobeans** [CreateJS/EaselJS#678]
So I have a tile map which after its read from a JSON file, it creates a grid of Bitmap objects and puts them into a Stage.

When the camera is totally upright is looks ok, but as soon as the stage rotates (rotating the camera is an essential part of my game) you see these lines that break up the tiles. I have tried putting the tiles into a separate Container and then caching it. Still breaks up.

Here's a pic: http://imgur.com/yT854vt

Any ideas?

Thank you
-Adam

**Comment by krazbobeans** [comment]
Here's another screenshot where the tile breaking looks bad
http://imgur.com/pELULJ0

Thanks again

**Comment by lannymcnie** [comment]
This looks like an antialiasing issue, and is browser related. I might recommend making your tiles a little larger than they need to be, and overlapping them.

**Comment by krazbobeans** [comment]
Ah ok, thanks. I was thinking maybe just scale them by 1.01 so they didn't have to shift positions.

Also, I tried to put all the tiles into one container and cache it since the tiles themselves don't animate. That's supposed to improve performance right? When I did that it made the framerate drop from 58 to like 7. I only called "cache" on the container one time after the tiles were loaded from the JSON. Any thoughts?

Thanks
